mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule LoggerLagerBackend.Mixfile do
 
   defp deps do
     [
-      {:lager, "~> 3.2"},
+      {:lager, ">= 3.2.0 and < 4.0.0"},
```

Comment:
According to https://hexdocs.pm/elixir/Version.html#module-requirements, this is exactly equivalent to the original expression (`~> 3.2`). Any reason to change this?